Services/Block/classes/class.ilBlockGUI.php

```diff
@@ -876,22 +886,22 @@ public function getHTMLNew(): string
         switch ($this->getPresentation()) {
             case self::PRES_SEC_LEG:
                 $panel = $factory->panel()->secondary()->legacy(
-                    $this->getTitle(),
+                    $this->specialCharsAsEntities($this->getTitle()),
```

Comment:
One can debate about if this (and the other five changed lines) belongs to `ilBlockGUI` or the respective UI component, but there is already some similar "title handling/string transformation" in `ilBlockGUI`.